eng/DefaultGenApiDocIds.txt

```diff
@@ -27,7 +27,6 @@ T:System.Runtime.CompilerServices.IteratorStateMachineAttribute
 T:System.Runtime.CompilerServices.MethodImpl
 T:System.Runtime.CompilerServices.NullableAttribute
 T:System.Runtime.CompilerServices.NullableContextAttribute
-T:System.Runtime.CompilerServices.PreserveDependencyAttribute
```

Comment:
Should we add `DynamicDependency` here? That way the `ref` assemblies won't start getting it when someone re-runs the generator.